src/infrastructure/chatwoot/sync_pg_integration_test.go

```
@@ -0,0 +1,258 @@
+	if err := db.PingContext(ctx); err != nil {
+		t.Fatalf("ping %s: %v", pgIntegrationEnv, err)
+	}
+	return db, dsn
+}
+
+// truncateChatwootData clears the rows an import writes while leaving the
+// account, inbox and agent fixture in place, so each test starts from a known
+// empty inbox without re-running the migrations.
+func truncateChatwootData(t *testing.T, db *sql.DB) {
+	t.Helper()
+	if _, err := db.Exec(`TRUNCATE messages, conversations, contact_inboxes, contacts RESTART IDENTITY CASCADE`); err != nil {
```

golangci-lint (2.13.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)